pkg/api/v1/types.go

```diff
@@ -668,6 +668,11 @@ type EmptyDirVolumeSource struct {
 	// More info: http://kubernetes.io/docs/user-guide/volumes#emptydir
 	// +optional
 	Medium StorageMedium `json:"medium,omitempty" protobuf:"bytes,1,opt,name=medium,casttype=StorageMedium"`
+	// Total amount of local storage required for this directory.
+	// The default is nil which means that the directory can use all available local storage on the node.
```

Comment:
Don't document the default as "nil" (that's a go-ism).  Instead document as "If not specified, the default is to not enforce a limit".  Though we might want to loosen this and say "If not specified, the default limit is implementation defined".  Default tmpfs is (I think) 1/2 physical mem.  Default disk is all storage, etc.  I don't think we need to define that here, though.